service-front/module/Application/src/Controller/CPFlowController.php

```diff
@@ -569,16 +569,15 @@ public function enterAddressManualAction(): ViewModel|Response
         $uuid = $this->params()->fromRoute("uuid");
         $detailsData = $this->opgApiService->getDetailsData($uuid);
 
-        $form = $this->createForm(CpAltAddress::class);
-        $form->setData($detailsData['alternateAddress'] ?? []);
+        $form = $this->createForm(Address::class);
```

Comment:
i think because you've removed the line: `$form->setData($detailsData['alternateAddress'] ?? []);` then when you go to the next page the address is not auto-populated with the info from the selected address on this page.